### PR TITLE
Fix voice-assistant display configs

### PR DESCRIPTION
**Pull request by deisterhold**
Adds `invert_colors: false` to display component so that firmware will build.

### CODE REVIEW

**Comment by LiamAkkerman** [comment]
I have also run into this issue since https://github.com/esphome/esphome/pull/7292 was merged. Just as a note, this PR only addresses the ESP32-S3-BOX yaml, not the ESP32-S3-BOX-3 or ESP32-S3-BOX-Lite yamls.

**Comment by deisterhold** [comment]
Updated to include the other two variants as well.

**Comment by LiamAkkerman** [comment]
Using my ESP32-S3-BOX-3, I can confirm that making the following change allowed me to build again and the voice assistant functions
```
packages:
  esphome.voice-assistant: github://deisterhold/firmware/voice-assistant/esp32-s3-box-3.yaml@patch-1
  # esphome.voice-assistant: github://esphome/firmware/voice-assistant/esp32-s3-box-3.yaml@main
```

File: voice-assistant/esp32-s3-box-lite.yaml [hunk 1]
```diff
@@ -500,6 +500,7 @@ display:
     dc_pin: 4
     reset_pin: 48
     update_interval: never
+    invert_colors: false
```
Comment (jesserockz) [inline voice-assistant/esp32-s3-box-lite.yaml:503]: Please don't add changes to a PR if you have not tested it
```suggestion
    invert_colors: true
```

**Review by jesserockz — CHANGES_REQUESTED** [review]

**Pull request merged**